src/server/server.ts

```diff
@@ -55,6 +55,14 @@ export const createServer = (
     });
   });
 
+  app.get("/:userPoolId/.well-known/openid-configuration", (req, res) => {
+    res.status(200).json({
+      id_token_signing_alg_values_supported: ["RS256"],
+      jwks_uri: `http://localhost:9229/${req.params.userPoolId}/.well-known/jwks.json`,
+      issuer: `http://localhost:9229/${req.params.userPoolId}`,
```

Comment:
I _think_ this needs additional items, such as:
```
"authorization_endpoint": string,
"id_token_signing_alg_values_supported": string[],
"issuer": string,
"jwks_uri": string,
"response_types_supported": string[],
"scopes_supported": string[],
"subject_types_supported": string[],
"token_endpoint": string,
"token_endpoint_auth_methods_supported": string[],
"userinfo_endpoint": string,
```

Reply:
And as a minor request, the domain should probably just be whatever the request domain is (with this being a local setup, could be coming from a container).

Reply:
Can i close this conversation or would you suggest to implement those additional fields?